components/Datasets/AdminUpdateDatasetPage.vue

```diff
@@ -73,4 +139,17 @@ const save = async () => {
     loading.value = false
   }
 }
+
+async function deleteDataset() {
+  loading.value = true
+  try {
+    await $api(`/api/1/datasets/${route.params.id}`, {
+      method: 'DELETE',
+    })
+    navigateTo(`${config.public.apiBase}/en/beta/admin/organizations/${route.params.oid}/datasets`, { external: true })
```

Comment:
The correct redirect logic should be something like this : https://github.com/datagouv/front-end/blob/feat/remove-objects-edit-mode/pages/beta/admin.vue#L71

apiBase is for api or links not already moved to this new front-end

Reply:
thanks indeed it is better